### ISSUE TITLE
Incorrect Max amount in Withdraw Liquidity&Reward section

### ISSUE THREAD

**Issue by Mitra8** [AcalaNetwork/acala-dapp#214]
Incorrect Max amount when withdrawing liquidity in Withdraw Liquidity&Reward section. In the example below, actual renBTC max amount is 0.000792, however 'the left part' shows 7.959574
![image](https://user-images.githubusercontent.com/69203809/89385970-1a0da200-d709-11ea-89fe-0abca5a37d6a.png)

5H985QyVoRZudfw2p6DvmwhdfYob8LkgK4pCZdf9rnioUZUd

**Comment by qwer951123** [comment]
The 7.959575 is not your renTBC amount, it is your dex pool shares

**Comment by Mitra8** [comment]
It shows below that the share is 0.000384%, why 7.959574?

**Comment by qwer951123** [comment]
> It shows below that the share is 0.000384%, why 7.959574?

the actually share numerical count is 7.959574, the 0.000384 means the proportion of the whole shares

**Comment by Mitra8** [comment]
In what units this 7.959574 is measured? Are these "Pool shares" something synthetic and are they internally calculated for each pool?
I performed another test for renBTC + aUSD pool to show is my logic:
1. In Deposit Liquidity tab I add 0.04 renBTC and corresponding amount in aUSD at the exchange rate at the time of the transaction.
![image](https://user-images.githubusercontent.com/69203809/89452102-d2693380-d765-11ea-9f84-54e7b770a3ef.png)
The deposit in aUSD equivalent is:
![image](https://user-images.githubusercontent.com/69203809/89452264-0a707680-d766-11ea-98d4-ac437a8cb00c.png)

2. In Withdraw Liquidity & Reward tab I see Pool shares 405.827886 and the same Max amount 405.827886. On the right side there are actually the same amounts as were deposited: 0.039999 renBTC + 462.574342 aUSD.
![image](https://user-images.githubusercontent.com/69203809/89452600-98e4f800-d766-11ea-9433-db79ceffbf0e.png)
In the Pool summary, it shows 0.019432% share, which I could clearly recalculate:
![image](https://user-images.githubusercontent.com/69203809/89452862-02fd9d00-d767-11ea-9362-4c0144e7aa13.png)

So I just don't understand what these "Pool shares" with '405.827886 amount' mean. If it's something technical for Polkadot language, it may not be so clear for ordinary users.

**Comment by Mitra8** [comment]
Now I see that in Polkadot interface:
![image](https://user-images.githubusercontent.com/69203809/89519933-cfae2300-d7e5-11ea-9c03-d686415b0324.png)
So recalculation shows:
![image](https://user-images.githubusercontent.com/69203809/89520206-2e739c80-d7e6-11ea-8a2b-f7e65f200014.png)
However, I guess it's not so user friendly and another presentation for UI would be more relevant.

**Comment by qwer951123** [comment]
Right, This is not clear for the user, we will make some improvements on it.
